conf/custom/nml.config

```diff
@@ -1,13 +1,9 @@
-// Cores
-params {
```

Comment:
Are all these thread parameters being removed as they're now being handled by a different controller?

Reply:
Using `task.cpus` and `task.cpus * 2` now for the different processes so no matther what the resources given are, the correct threads value is input for that process